app/Http/Controllers/BookController.php

```diff
@@ -21,13 +21,24 @@ public function index()
     public function detail($slug)
     {
         try {
+            $liked = false;
             $book = Book::where('slug', $slug)->firstOrFail();
+            if (Auth::check()) {
+                $isLiked = $book->likedUsers()->where('user_id', Auth::id())->exists();
```

Comment:
```suggestion
                $isLiked = $book->likedUsers->wherePivot('user_id', Auth::id())->exists();
```
dùng cái này thử xem